src/site/_filters/gitlocalize-link.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require("path");
+const site = require("../_data/site");
+
+module.exports = (inputPath, lang) => {
+  if (!inputPath) {
+    /* eslint-disable-next-line */
+    console.warn(
```

Comment:
This is presumably about to crash. Do we expect this to happen?
If so, then we should return an empty string etc—otherwise I'm happy to remove this as the `Error` itself is notable enough.

Reply:
Empty string should be ok